fe/fe-core/src/main/java/org/apache/doris/clone/DynamicPartitionScheduler.java

```diff
@@ -351,17 +348,12 @@ private ArrayList<AddPartitionClause> getAddPartitionClause(Database db, OlapTab
      */
     private void setStorageMediumProperty(HashMap<String, String> partitionProperties,
             DynamicPartitionProperty property, ZonedDateTime now, int hotPartitionNum, int offset) {
-        if ((hotPartitionNum <= 0 || offset + hotPartitionNum <= 0) && !property.getStorageMedium()
+        if (hotPartitionNum <= 0 || offset + hotPartitionNum <= 0 || property.getStorageMedium()
                 .equalsIgnoreCase("ssd")) {
             return;
         }
-        String cooldownTime;
-        if (property.getStorageMedium().equalsIgnoreCase("ssd")) {
-            cooldownTime = TimeUtils.longToTimeString(DataProperty.MAX_COOLDOWN_TIME_MS);
```

Comment:
why remove this line?

Reply:
mistook assume that no cooldown time equals max cooldown time.   rollback this function.